framework/core/src/Extension/Bisect.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Extension;
+
+use Closure;
+use Composer\Command\ClearCacheCommand;
+use Flarum\Settings\SettingsRepositoryInterface;
+use RuntimeException;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\NullOutput;
+
+class Bisect
+{
+    protected BisectState $state;
+    protected bool $break = false;
```

Comment:
Could you add a comment explaining that `break` is present for when we will be running the bisecter across multiple instances of processes (i.e. requests), if that's a correct understanding of why we need it?